rpc/rpc-wrappers.go

```diff
@@ -372,6 +374,23 @@ func NewRegistrationAuthorityServer(rpc Server, impl core.RegistrationAuthority,
 		return
 	})
 
+	rpc.Handle(MethodDeactivateAuthorization, func(ctx context.Context, req []byte) (response []byte, err error) {
```

Comment:
Why named returns here? I think explicit returns would be better for this function.